modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/resource/v1_0/WishListResourceImpl.java

```diff
@@ -143,35 +142,44 @@ public WishList postChannelWishList(
 				_serviceContextHelper.getServiceContext(
 					commerceChannel.getSiteGroupId()));
 
-		_postWishListItems(
-			commerceWishList, accountId, wishList.getWishListItems());
+		_updateNestedResources(
+			accountId, commerceWishList, wishList.getWishListItems());
 
 		return _toWishList(commerceWishList);
 	}
 
-	private void _postWishListItems(
-			CommerceWishList commerceWishList, Long accountId,
-			WishListItem[] wishListItems)
+	private Long _getCommerceAccountId(
+			Long accountId, CommerceChannel commerceChannel)
 		throws Exception {
 
-		if (ArrayUtil.isEmpty(wishListItems)) {
-			return;
-		}
-
-		_wishListItemResource.setContextAcceptLanguage(contextAcceptLanguage);
```

Comment:
We are making new endpoint like this to implicitly test nested resources endpoints and call, please revert to this version